selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/configuration/SeleniumTestConfiguration.java

```diff
@@ -128,12 +128,6 @@ public boolean test(Map.Entry<K, V> entry) {
       name = name.replace('_', '.');
       name = name.replace("=", "_");
 
-      // convert value of CHE_INFRASTRUCTURE to upper case to comply with Infrastructure
```

Comment:
As now, the `che.infrastructure` can be defined also as system property, I had to move this logic somewhere else (`InMemoryTestConfiguration`). If the new place is not the best one, please let me know.